views/_includes/layouts/base.html

```diff
@@ -20,7 +20,8 @@
                     </a>
                     <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
                         <div class="navbar-nav">
-                            <a class="nav-item nav-link active" href="/">Home<span class="sr-only">(current)</span></a>
+                            
+                            <a class="nav-item nav-link active" href="/job-roles">Job Roles<span class="sr-only">(current)</span></a>
```

Comment:
`(current)` is a good idea for screenreaders, but needs to be dynamically applied